### PR TITLE
chore: replace readable-stream with built-in stream

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/pinojs/pino-pretty/issues/541

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -40,7 +40,7 @@ jobs:
       contents: read
     strategy:
       matrix:
-        node-version: [18.17, 20, 22]
+        node-version: [18.19, 20, 22]
```
Comment (kibertoad) [inline .github/workflows/ci.yml:43]: isn't that technically a semver major? I'd rather use the opportunity to drop node 18 from the list altogether

**Comment by mcollina** [comment]
@kibertoad we just bumped the major, I don't think we need another bump.

**Comment by mcollina** [comment]
cc @jsumners

**Comment by jsumners** [comment]
We probably should have done it then, but I'm really not opposed to version churn on this module. I consider this module non-essential, and as such don't think it matters too much how often it is released.

**Pull request merged**